### PR TITLE
Add logging and replace print statements Fixes #13

### LINKED ISSUE #13: Logging
Swap the warnings package out with the logging package, which will also replace any print statements with better logging methods for INFO, DEBUG, WARNING, and ERROR logging.

### PR DESCRIPTION
**Pull request by MohamedNasser8**

### CODE REVIEW

File: manage.py [hunk 1]
```diff
@@ -1,6 +1,11 @@
 import os
 import sys
 import venv
+import logging
+
+#logging root configuration
+logging.basicConfig(level=logging.DEBUG, # This will log all levels of messages
```
Comment (ltorres6) [inline manage.py:7]: Logging should be INFO by default here to prevent a ton of information on the console.

File: manage.py [hunk 2]
```diff
@@ -13,14 +18,18 @@ def distribute():
...
-
+    create_venv()
     install()
 
     os.system(activate() + ' && ' + 'pip install --upgrade build')
     os.system(activate() + ' && ' + 'python -m build')
     os.system(activate() + ' && ' + 'pip install --upgrade twine')
     os.system(activate() + ' && ' + 'twine upload dist/*')
 
+def create_venv():
```
Comment (ltorres6) [inline manage.py:29]: Add a snippet here that does nothing if you already have an existing environment please.

**Review by ltorres6 — CHANGES_REQUESTED** [review]
Thanks for the submission! I've left some comments.

File: manage.py (hunk 2 above)
Comment (MohamedNasser8) [reply manage.py:29]: I modified it.

**Comment by MohamedNasser8** [comment]
I think it's now resolved.

File: manage.py [hunk 3]
```diff
@@ -1,6 +1,11 @@
 import os
 import sys
 import venv
+import logging
+
+#logging root configuration
+logging.basicConfig(level=logging.INFO, # This will log all levels of messages
```
Comment (ltorres6) [inline manage.py:7]: One more comment here. This setting will  avoid logging any debug messages, so it does not log all levels of messages.

Comment (MohamedNasser8) [reply manage.py:7]: I think when need to debug,  set the debug level to DEBUG and return it to default after end debugging.
something like this
`    if debug_mode:
        previous_level = logging.getLogger().level  # Store current level
        logging.getLogger().setLevel(logging.DEBUG)  # Set to DEBUG for this block
        logging.debug(debug_message)
        logging.getLogger().setLevel(previous_level)`

Comment (MohamedNasser8) [reply manage.py:7]: I will have variable debug_mode which it's default is false but when it's true i will change the Level to Debuger.

**Comment by MohamedNasser8** [comment]
I will have variable debug_mode which it's default is false but when it's true i will change the Level to Debuger.

**Review by ltorres6 — COMMENTED** [review]
LGTM

**Review by ltorres6 — APPROVED** [review]
LGTM

**Pull request merged**